package.json

```diff
@@ -40,13 +39,10 @@
     "rc-slider": "^9.2.2",
     "react": "^16.12.0",
     "react-bootstrap": "1.0.1",
-    "react-bootstrap-table": "^4.3.1",
```

Comment:
So react-bootstrap-table is being used currently in `Admin.js` for the admin table.

Reply:
thanks for the catch!